components/ReactBKKLogo/RevealEllipse.js

```diff
@@ -44,6 +45,15 @@ class RevealEllipse extends React.Component {
       this.timer = Date.now()
       this.raf = window.requestAnimationFrame(this.revealStep)
     }, this.props.delay)
+    document.getElementsByTagName('BODY')[0].addEventListener('click', function (e) {
+      if (e.target && e.target.tagName === 'SPAN') {
+        scroller.scrollTo(e.target.textContent, {
```

Comment:
We should not depend on the text of the link, because otherwise link scrolling will break if we change the text.